pom.xml

```diff
@@ -81,7 +81,7 @@
         <dependency>
             <groupId>nl.knaw.dans.lib</groupId>
             <artifactId>dans-dataverse-scala-lib_2.12</artifactId>
-            <version>1.1.0</version>
+            <version>1.1.1-SNAPSHOT</version>
```

Comment:
Why change this to a snapshot dependency?

Reply:
sorry, I had changed it to test something else, and forgot to change it back.

Reply:
Please, change it back then.

Reply:
done